src/Main.elm

```diff
@@ -36,8 +45,11 @@ update msg model =
 view : Model -> Html Msg
 view model =
     div []
-        [ img [ src "/logo.svg" ] []
-        , h1 [] [ text "Your Elm App is working!" ]
+        [ button [ onMouseDown (NoteOn C), onMouseUp NoteOff ] [ text "C" ]
+        , button [ onMouseDown (NoteOn DSharp), onMouseUp NoteOff ] [ text "D#" ]
+        , button [ onMouseDown (NoteOn FSharp), onMouseUp NoteOff ] [ text "F#" ]
+        , button [ onMouseDown (NoteOn A), onMouseUp NoteOff ] [ text "A" ]
+        , text (Maybe.withDefault "" (Maybe.map toString model))
```

Comment:
Its a bit early to refactor this but it would be nice to have the text in it's own paragraph. I am a big fan of semantic markup.

Reply:
Sure, but this is more like a debug output... Will be removed eventually.